### PR TITLE
Add CONTRIBUTING.md changes for Ubuntu users to make debugging more…

### PR DESCRIPTION
**Pull request by sylphiae**
…clear

### What github issue is this PR for, if any?
N/A

### What does this code do, and why?
Updates CONTRIBUTING.md for Ubuntu users to help streamline local installations of arooo. 

### How is this code tested?
It's just a change to documentation; it is not a code change so tests shouldn't be affected.

### Are any database migrations required by this change?
N/A

### Are there any configuration or environment changes needed?
N/A

### Screenshots please :)
I think the diff will show my changes better than a screenshot will. There were 4 changes to CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,10 +54,96 @@ $ cp config/application.example.yml config/application.yml
...
+
+### Steps to run arooo server locally
+
+1. `bundle exec rake populate:users` to set up dummy data
+1. `bundle exec rails server` don't forget to use http://localhost:3000 and not https
+1. `bundle exec rails console` Optional - useful for looking at and changing your local data)
+
+### Docker setup (optional)
+
+1. Install docker and docker compose# Contributing Guide
```
Comment (anaulin) [inline CONTRIBUTING.md:81]: Perhaps you accidentally copy+pasted the entire document back into itself at this point? It seems most of it duplicated starting from here on down.

**Review by anaulin — APPROVED** [review]
Thank you for improving the documentation!

Looks like there is some duplicate content in the diff. Once that is fixed, feel free to merge.

File: CONTRIBUTING.md (hunk 1 above)
Comment (sylphiae) [reply CONTRIBUTING.md:81]: Hmm, I didn't think I copy-pasted anything. I deleted a line and this is how Git decided to handle the deleted line I think.

Comment (sylphiae) [reply CONTRIBUTING.md:81]: Oh no wait, you're right there is a duplicate. I should fix this.

**Review by brittag — APPROVED** [review]
Looks good to me too!

**Pull request merged**